server/src/configs/dbHelper.js

```diff
@@ -0,0 +1,35 @@
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import mongoose from 'mongoose';
+
+let mongod;
+async function connect() {
+    mongod = new MongoMemoryServer();
```

Comment:
This memory server is good for testing but not for real time applications, you have to use a mongodb which is hosted at some place.

Reply:
yes, this is for testing alone. db url is stored in .env file. i will rename the file.